src/interfaces/cVoxelType.ts

```diff
@@ -93,3 +93,465 @@ export const cVoxelTest: CVoxelVisType[] = [
     scale: 1,
   },
 ];
+
+export const initCVoxel: CVoxel[] = [
```

Comment:
pls move this constants definition to `constants` directory

Reply:
I moved to the directory.